lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/dao/MessageUtils.java

```diff
@@ -18,8 +18,10 @@
 import org.sagebionetworks.repo.model.message.MessageStatus;
 import org.sagebionetworks.repo.model.message.MessageStatusType;
 import org.sagebionetworks.repo.model.message.MessageToUser;
+import org.sagebionetworks.util.ValidateArgument;
 
 public class MessageUtils {
+	public static final int MAX_LENGTH = 1024;
```

Comment:
Please consider moving this length constant to DBOMessageToUser.  Look for a way to avoid defining the constant twice.